pkg/action/action_handler.go

```diff
@@ -46,9 +46,10 @@ type ActionHandlerConfig struct {
 	kubeletClient  *kubeclient.KubeletClient
 	StopEverything chan struct{}
 	sccAllowedSet  map[string]struct{}
+	cAPINamespace  string
 }
 
-func NewActionHandlerConfig(kubeClient *client.Clientset, kubeletClient *kubeclient.KubeletClient, sccSupport []string) *ActionHandlerConfig {
+func NewActionHandlerConfig(cApiNamespace string, kubeClient *client.Clientset, kubeletClient *kubeclient.KubeletClient, sccSupport []string) *ActionHandlerConfig {
```

Comment:
Maybe move the **cApiNamespace** to the last of the function parameter?

Reply:
I prefer to have it as a first one. The emphasis is important, IMHO.